test/jasmine/tests/image_test.js

```diff
@@ -535,4 +667,42 @@ describe('image hover:', function() {
             .then(done);
         });
     });
+
+    describe('for `image_labuda_droplets_source` defined by source', function() {
+        var mock = require('@mocks/image_labuda_droplets_source.json');
+        beforeAll(function() {
+            gd = createGraphDiv();
+        });
+
+        afterAll(destroyGraphDiv);
+
+        function _hover(x, y) {
+            var evt = { xpx: x, ypx: y };
+            return Fx.hover('graph', evt, 'xy');
+        }
+
+        [
+          ['x', '205'],
+          ['y', '125'],
+          ['color', '[78, 77, 83, 1]'],
+          ['color[0]', '78'],
+        ].forEach(function(test) {
+            it('should support hovertemplate variable ' + test[0], function(done) {
+                var mockCopy = Lib.extendDeep({}, mock);
+                mockCopy.data[0].colormodel = 'rgba';
+                mockCopy.data[0].hovertemplate = '%{' + test[0] + '}<extra></extra>';
+                Plotly.newPlot(gd, mockCopy)
+                .then(delay(100)) // wait for the image to be loaded into canvas
```

Comment:
This is needed because `Plotly.newPlot` will resolve even if the promise in the `plot` step is not completed. cc @alexcjohnson

Reply:
The delay is not necessary anymore with and has been removed in e4b923b9c043f71f98dde8de38bd830f96db34b5